spaces/S000178/properties/P000147.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000178
+property: P000147
+value: true
+refs:
+  - doi: 10.1016/0016-660X(72)90026-8
+    name: A topological view of P-spaces (A. Misra)
+---
+
+Shown in Example 3.1 of {{doi:10.1016/0016-660X(72)90026-8}}.
```

Comment:
Possible justification: It's a subspace of S177 and the P-space property is preserved by subspaces.